src/AppServices/KafkaPageV3.tsx

```diff
@@ -0,0 +1,307 @@
+            <Title headingLevel="h2">{t("kafkaInstanceCapacityTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Stack hasGutter>
+              <Text component={TextVariants.p}>
+                {t("kafkaInstanceCapacityDescription")}
+              </Text>
+              <KafkaInstanceCapacityTable />
+            </Stack>
+          </CardBody>
+          <CardFooter>
+            <TextContent className={"pf-u-font-size-sm"}>
+              <Trans
+                ns={"kafkaoverview-v3"}
+                i18nKey={"kafkaInstanceCapacityfooter"}
+                components={[<SupportLink />, <MonitoringDiskSpaceLink />]}
+              />
+            </TextContent>
+          </CardFooter>
+        </Card>
+      </MarketingPageSection>
+      <MarketingPageSection
+        className={"kafka-overview--page-section--marketing"}
+      >
+        <Card>
+          <CardTitle>
+            <Title headingLevel="h2">{t("cloudProvidersTitle")}</Title>
+          </CardTitle>
+          <CardBody>
+            <Flex spaceItems={{ default: "spaceItemsXl" }}>
+              <Flex
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Split hasGutter>
+                    <SplitItem>
+                      <img src={LogoAWS} alt={""} style={{ height: "60px" }} />
+                    </SplitItem>
+                    <SplitItem>
+                      <Title headingLevel="h2" className={"pf-u-pt-sm"}>
+                        {t("awsWebServiceTitle")}
+                      </Title>
+                    </SplitItem>
+                  </Split>
+                </FlexItem>
+              </Flex>
+              <Divider isVertical />
+              <Flex
+                justifyContent={{ default: "justifyContentCenter" }}
+                flex={{ default: "flex_1" }}
+              >
+                <FlexItem>
+                  <Split hasGutter>
+                    <SplitItem>
+                      <img
+                        src={LogoAzure}
+                        alt={""}
+                        style={{ height: "60px" }}
+                      />
+                    </SplitItem>
+                    <SplitItem>
+                      <Title headingLevel="h2" className={"pf-u-pt-sm"}>
+                        {t("microsoftAzureTitle")}
+                      </Title>
+                      <Text
+                        component={TextVariants.p}
+                        className={"pf-u-color-200"}
+                      >
+                        {t("microsoftAzureDescription")}
+                      </Text>
+                    </SplitItem>
+                  </Split>
+                </FlexItem>
+              </Flex>
+            </Flex>
+          </CardBody>
+        </Card>
+      </MarketingPageSection>
+    </>
+  );
+};
+
+const SupportLink: FunctionComponent = ({ children }) => (
```

Comment:
There is a shared component for creating external links https://redhat-developer.github.io/app-services-ui-components/?path=/story/shared-externallink--external-link. I think we can make use of that instead of creating new components for each external link